src/test/java/com/kodilla/ecommercee/entity/UserOrderRepositoryTest.java

```diff
@@ -66,13 +69,9 @@ public void testOrderSave() {
         assertEquals(3, cartCount);
 
         // Clean
-        orderRepository.deleteById(orderOne.getId());
-        orderRepository.deleteById(orderTwo.getId());
-        orderRepository.deleteById(orderThree.getId());
-        cartRepository.deleteById(cartOne.getId());
-        cartRepository.deleteById(cartTwo.getId());
-        cartRepository.deleteById(cartThree.getId());
-        userRepository.deleteById(user.getId());
+        orderRepository.deleteAll();
```

Comment:
Ogólnie ok bo to tylko testy, ale jeśli testy byłyby robione na prawdziwej bazie to usuniesz wszytko, co w niej jest. A tego raczej byśmy nie chceli ....

Reply:
poprawione